test-frame-common/src/main/java/io/skodjob/testframe/clients/KubeClient.java

```diff
@@ -32,4 +34,44 @@ public KubernetesClient getClient() {
     public OpenShiftClient getOpenShiftClient() {
         return client.adapt(OpenShiftClient.class);
     }
+
+    public String getKubeconfigPath() {
+        return kubeconfigPath;
+    }
+
+    private Config getConfig() {
+        if (TestFrameEnv.KUBE_USERNAME != null
+                && TestFrameEnv.KUBE_PASSWORD != null
+                && TestFrameEnv.KUBE_URL != null) {
+            Exec.exec(Arrays.asList("oc", "login", "-u", TestFrameEnv.KUBE_USERNAME,
+                    "-p", TestFrameEnv.KUBE_PASSWORD,
+                    "--insecure-skip-tls-verify",
+                    "--kubeconfig", TestFrameEnv.USER_PATH + "/test.kubeconfig",
+                    TestFrameEnv.KUBE_URL));
+            kubeconfigPath = TestFrameEnv.USER_PATH + "/test.kubeconfig";
+            return new ConfigBuilder()
+                    .withUsername(TestFrameEnv.KUBE_USERNAME)
+                    .withPassword(TestFrameEnv.KUBE_PASSWORD)
+                    .withMasterUrl(TestFrameEnv.KUBE_URL)
+                    .withDisableHostnameVerification(true)
+                    .withTrustCerts(true)
+                    .build();
+        } else if (TestFrameEnv.KUBE_URL != null
+                && TestFrameEnv.KUBE_TOKEN != null) {
+            Exec.exec(Arrays.asList("oc", "login", "--token", TestFrameEnv.KUBE_TOKEN,
+                    "--insecure-skip-tls-verify",
+                    "--kubeconfig", TestFrameEnv.USER_PATH + "/test.kubeconfig",
+                    TestFrameEnv.KUBE_URL));
+            kubeconfigPath = TestFrameEnv.USER_PATH + "/test.kubeconfig";
+            return new ConfigBuilder()
+                    .withOauthToken(TestFrameEnv.KUBE_TOKEN)
+                    .withMasterUrl(TestFrameEnv.KUBE_URL)
+                    .withDisableHostnameVerification(true)
+                    .withTrustCerts(true)
+                    .build();
```

Comment:
should here be check for the `CLIENT_TYPE`? Because otherwise I think it will throw and exception in case that `oc` is not installed